components/gfx/display_list/mod.rs

```diff
@@ -1043,6 +1043,22 @@ impl ClippingRegion {
     }
 }
 
+impl fmt::Debug for ClippingRegion {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if self.main == max_rect() && self.complex.is_empty() {
```

Comment:
I considered this, but I'm not sure if it is actually more readable. Is your concern that I'm doing the equality check more than once?